src/test/codegen-units/item-collection/implicit-panic-call.rs

```diff
@@ -0,0 +1,58 @@
+// compile-flags:-Zprint-mono-items=eager
```

Comment:
Wouldn't the eager mode consider `panic` function to be one of roots and hide the bug?

Reply:
Fixed. Good catch!